DECISIONLOG.md

```diff
@@ -8,4 +8,20 @@ In the `int_netsuite__transactions_with_converted_amounts` model, account types
 
 ## Creation of 'Other' Account Category for Non Posting and Statistical Account Types
 
-As mentioned above, in the `int_netsuite__transactions_with_converted_amounts`/`int_netsuite2_tran_with_converted_amounts` models, account types are bucketed into broader account _categories_. There is no standard category for `Non Posting` and `Statistical` account types and transactions from these kinds of accounts are excluded from all financial reports. However, they are used for other workflows in Netsuite, so we have bucketed `Non Posting` and `Statistical` account types into a new `Other` account category.
\ No newline at end of file
+As mentioned above, in the `int_netsuite__transactions_with_converted_amounts`/`int_netsuite2_tran_with_converted_amounts` models, account types are bucketed into broader account _categories_. There is no standard category for `Non Posting` and `Statistical` account types and transactions from these kinds of accounts are excluded from all financial reports. However, they are used for other workflows in Netsuite, so we have bucketed `Non Posting` and `Statistical` account types into a new `Other` account category.
+
+## Why converted transaction amounts are null if they are non-posting
+
+In our `intermediate` Netsuite models, we translate amounts from posted transactions into their proper `converted_amount` values based on the exchange rates in the reporting and transaction periods. That way, customers will always have accurate `converted_amount` data that can help them validate their financial reporting.
+
+For the sake of financial fidelity, we decided not to convert amounts that are non-posting because the exchange rates are subject to change. While that can provide additional value for customers looking to do financial forecasting, we do not want to create confusion by bringing in converted transactions that have amounts that are variable to change, and disrupt existing financial reporting processes.
+
+For customers interested in creating future-facing `converted_amount` values, our recommendation would be to materialize the `intermediate` tables to grab the exchange rate data in your internal warehouse, then leverage the `transaction_amount` in these particular cases to produce the future `converted_amounts`.
```

Comment:
> "our recommendation would be to materialize the `intermediate` tables"

Are these currently ephemeral by default and we are suggesting customers materialize them as tables or views? if so, i'd update this wording to be "our recommendation would be to materialize the `intermediate` models as views or tables (they are currently ephemeral by default)..."

or better yet, it could be good to provide the `dbt_project.yml` code snippet for how to do so ^

Reply:
@fivetran-avinash Could you take a look at Jamie's comment since this came from your PR?

Reply:
Thanks @fivetran-jamie! I agree with your suggestions.

@fivetran-catfritz 

The code snippet they would want to update in the `dbt_project.yml` would be underneath `models`, so something like: 

> For customers interested in creating future-facing `converted_amount` values, our recommendation would be to materialize the `intermediate` tables to grab the exchange rate data in your internal warehouse, then leverage the `transaction_amount` in these particular cases to produce the future `converted_amounts`. You can update the models section of your `dbt_project.yml` to update the materialization.
> 
> ```
>     netsuite2:
>       intermediate:
>         +materialized: [table or view]
> ```
> 

Should suffice to add in the DECISIONLOG.

Reply:
Updated!